### ISSUE TITLE
Version 2.0.0 is not published in npm

### ISSUE THREAD

**Issue by jirutka** [alexgorbatchev/eslint-import-resolver-typescript#29]
There’s a tag [v2.0.0](https://github.com/alexgorbatchev/eslint-import-resolver-typescript/tree/v2.0.0), but the corresponding version is not available on npmjs.com.

**Comment by JounQin** [comment]
Yep, I haven't got the npm publish access for now. You can use [eslint-import-resolver-ts](https://github.com/rx-ts/eslint-import-resolver-ts) temporarily for now.

@alexgorbatchev

**Comment by jazzfog** [comment]
I think I broke my head today trying to apply `v2` instructions to `v1` code that got installed by `npm` :)

For now the easiest workaround is to install from the GitHub itself
`yarn add https://github.com/alexgorbatchev/eslint-import-resolver-typescript.git#2.0.0`

**Comment by JounQin** [comment]
@jazzfog The tagged commit is not up to date, I personally do not recommend doing like this. Try `eslint-import-resolver-ts` temporarily instead for now.

**Comment by jirutka** [comment]
So why the heck you've created the tag when you actually couldn't publish it and now you even said that it's not up to date? Moreover, the changelog contains only versions that don't exist at all. It's all very confusing.

**Comment by JounQin** [comment]
Not me. The repository owner is @alexgorbatchev .

**Comment by jirutka** [comment]
Oh, I see now, my apologies.

**Comment by alexgorbatchev** [comment]
All of the tests are failing for me with the following error:

```
❯ yarn test
yarn run v1.19.0
$ eslint -f friendly tests/**/*.{ts,tsx}


  ✘  https://google.com/#q=import%2Fno-unresolved

     Resolve error: typescript with invalid interface loaded as resolver


     tests/withoutPaths/index.ts:1:1
   > 1 | // import relative
       | ^
     2 | import './tsImportee'
     3 | import './tsxImportee'
     4 | import './dtsImportee'
```

Any idea what i'm missing? I havent seen this error before.

**Comment by JounQin** [comment]
@alexgorbatchev Sorry I forgot to add `pretest: 'r'` script, that's because the source is `.ts` based now, so a build step is required before `test`.

**Comment by alexgorbatchev** [comment]
published, there seem to be an issue with lodash #31
